.github/workflows/create-release.yml

```diff
@@ -283,24 +327,6 @@ jobs:
       env:
         BUILD_REMOVED: ${{ steps.build_diff.outputs.removed }}
         RUN_REMOVED: ${{ steps.run_diff.outputs.removed }}
-
-  run_if_packages_removed_with_force:
```

Comment:
I don't see where this has been moved to (or what it has been replaced with).

Do we still support the ability to force a release even if packages have been removed?

Reply:
As far as I could tell, this  particular job was ONLY checking if a package was removed with force, and printing a statement if it was.The logic to make sure we can force a release happens starting line 306 `- name: Fail If Packages Removed`, and I added  the print statement there if a package is removed with force! 

TLDR, this wasn't a super helpful code block and the core functionality you're asking about is still supported

Reply:
Gotcha. I see it now. Thanks for being patient with me 😄